about.html

```diff
@@ -74,36 +74,35 @@ <h2 class="font-bold uppercase text-gray-700 mt-6 mb-2">Pre koho to je?</h2>
                     <div class="text-3xl font-bold">Rodič</div>
                     <p>
                         Nájdi tú správnu súťaž pre tvoje dieťa,
-                        pochop tej ktorú už robí a maj
-                        prehľad o termínoch ktoré má.
+                        pochop tej, ktorú už robí a maj
+                        prehľad o termínoch, ktoré má dodržať.
                     </p>
                 </div>
             </div>
    
             <h2 class="font-bold uppercase text-gray-700 mt-6 mb-2">Čo všetko tu nájdem?</h2>
             <p>
-                Uvádzame termíny mnohých súťaží a akcií od organizácií ktoré sa sami na tvorbe
-                kalendára podieľajú. Väčšinou sa jedná o dlhodobé súťaže (semináre) alebo
-                sústredenia pre druhý stupeň základných škôl a stredné školy.
+                Uvádzame termíny mnohých súťaží a akcií od organizácií, ktoré sa sami na tvorbe
+                kalendára podieľajú. Akcie sú dlhodobé aj jednodňové, pre žiakov od 4. ročníka základnej školy po koniec strednej školy.
             </p>
 
             <div class="text-center mt-6 mb-2">
                 <a class="inline-block rounded bg-blue-500 text-white text-3xl py-2 px-6 font-bold" href="index.html">Prejsť na termíny</a>
             </div>
 
             <h2 class="font-bold uppercase text-gray-700 mt-16 mb-2">Chcem pomôcť, opraviť chybu alebo mám otázku</h2>
+            <p>
+                <b>Ak si našiel/-la chybu</b> môžeš nám napísať na
+                <a class="text-blue-500" href="mailto:kockaty.kalendar@p-mat.sk">kockaty.kalendar@p-mat.sk</a>
+                a my sa ju pokúsime opraviť. Ak si programátor/-ka, tak môžeš rovno otvoriť PR v
+                <a class="text-blue-500" href="https://github.com/kockatykalendar/data">repozitári s dátami</a>.
```

Comment:
Tu by som už nedával linky na repozitár, keďže je to hneď pod tým znovu.
Kľudne len, že "Ak si programator/-ka, tak môžeš rovno otvoriť PR na našom GitHub-e."

Čo myslíš?